openfecwebapp/templates/legal-advisory-opinions-landing.html

```diff
@@ -34,7 +34,7 @@
         <div class="row">
           <div class="message message--info">
             <h3>This feature is still in progress</h3>
-            <p>We are actively building the <span className="font-bold">advisory opinion search</span>, and it doesn&#39;t yet include some
+            <p>We are actively building the <strong class="font-bold">advisory opinion search</strong>, and it doesn&#39;t yet include some
```

Comment:
Sorry, I meant you can also remove `class="font-bold"`. The `strong` carries the styling as-is.